libs/designer/src/lib/ui/configuretemplate/workflows/selectWorkflows.tsx

```diff
@@ -66,16 +73,11 @@ export const SelectWorkflows = ({
     [onWorkflowsSelected]
   );
 
-  const differentIdThanResourceRecord = useMemo(() => {
-    const selectedDifferentNameWorkflowsList: Record<string, string | undefined> = {};
-    for (const workflow of Object.values(selectedWorkflowsList)) {
-      const workflowSourceId = workflow.manifest?.metadata?.workflowSourceId;
-      if (workflowSourceId && workflow.isManageWorkflow) {
-        selectedDifferentNameWorkflowsList[workflowSourceId] = workflow.id;
-      }
-    }
-    return selectedDifferentNameWorkflowsList;
-  }, [selectedWorkflowsList]);
+  const workflowSourceIdsInTemplate = useMemo(() => {
+    return Object.values(workflowsInTemplate)
+      .map((workflow) => workflow.manifest?.metadata?.workflowSourceId)
```

Comment:
I would suggest you to lowercase these, to do a case insensitive lookup..... since ids can be case insensitive